recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "e3sm_to_cmip" %}
-{% set version = "1.4" %}
+{% set version = "1.4.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: b552f78c30c33c0457cd89e541d485b9e0b5996a50e71d89650f21cb9f54ea1c
+  url: https://github.com/E3SM-Project/{{ name }}/archive/v.{{ version }}.tar.gz
```

Comment:
This change was due to a typo in the tag name and will likely need to be reverted in the next release.

Reply:
I edited the release to remove the offending ".", so I think you can change it back here.